react-design/my-app/src/Banner.jsx

```diff
@@ -0,0 +1,9 @@
+export default function Banner({ items }) {
+  function displayAnimal() {
```

Comment:
You will only want to display one item, not all of them. So this loop won't be necessary in the final design.